articles/introduction-to-reinforcement-learning/index.md

```diff
@@ -0,0 +1,179 @@
+
+They estimate their values on an infinite number of states and actions. This makes them suitable for handling high dimensional continuous action spaces.
+
+They are of two types:
+
+**Stochastic Policy Gradients (SPG)**
+They integrates over both the state and action spaces.
+
+**Deterministic Policy Gradients (DPG)**
+They integrates over only the state spaces.
+
+Due to these differences, SPG may need more samples to compute in cases of high dimensions action spaces.
+
+#### Actor-Critics (AC)
+![actor-critic](/engineering-education/introduction-to-reinforcement-learning/actor-critic.png)
+
+*[Image Source: Berkeley](http://rail.eecs.berkeley.edu/deeprlcourse-fa17/f17docs/lecture_5_actor_critic_pdf)*
+
+Actor-Critics (AC) method consists of two models: An Actor and A Critic.
+
+The actor learns the policy function while the critic learns the value function. They tends to learn both the policy and the value function.  
+
+The actor's role is to parameterize the selection of actions.
+The critic's role is to criticize whether the actions performed by the actor are good or bad. The actor then updates its parameters based on the critic’s appraisal.
+
+AC models tend to need much less training time compared to the PG methods.
+
+We can typically see some of AC models being used in the fields of both biology and psychology.
+
+### Applications of Reinforcement Learning
+#### Games
+The use of AI in building computer games has been increasing over the past couple of years.
+This is because games (by their very nature) provide varying challenges. This allows RL algorithms to showcase their weaknesses and/or strengths by playing games.
+
+In 2016, Deepmind Technologies built [AlphaGo](/https://deepmind.com/research/case-studies/alphago-the-story-so-far/). A RL controlled computer program that plays the board game, Go (an ancient Chinese board game). The computer program was able to beat Lee Sedol, a Korean Go world champion 4-1 in a 5 match game.
+
+Later versions of AlphaGo such as AlphaGo Zero and MuZero became even more powerful. They were completely self-taught using RL. This was significant as they were learning to play games without being taught the game rules.
+
+
+#### Robotics
+RL robots control Google’s data center to manage part of its [cooling infrastructure.](/https://deepmind.com/blog/article/safety-first-ai-autonomous-data-centre-cooling-and-industrial-control/)
+
+The AI robots exploits a similar approach to that used in AlphaGo, AlphaGo Zero and MuZero. This is by learning through trial and error.
+
+The algorithm worked by:
+1. Feeding it information gathered from the cooling data centers.
+2. Leaving it to determine which cooling configurations would reduce energy consumption.
+
+Google’s data center energy spending reduced by up to 40%.
+
+#### Trading and Finance
+RL agents are used to predict stock prices as well as predicting future sales. They can decide on whether to buy, hold or sell a particular stock.
```

Comment:
It would add much value to the article in adding some examples of such RL software used in trading and finance.